src/WebAppResolver.ts

```diff
@@ -33,7 +42,13 @@ export class WebAppResolver implements AppResourceResolver {
 
             await this.listWebAppsTask;
             const site = this.siteCache.get(nonNullProp(resource, 'id').toLowerCase());
-            return new ResolvedWebAppResource(subContext, nonNullValue(site));
+
+            const groupBy: string | undefined = getGlobalSetting<string>('groupBy', 'azureResourceGroups');
+            return new ResolvedWebAppResource(subContext, nonNullValue(site), {
+                // Multiple sites with the same name could be displayed as long as they are in different locations
+                // To help distinguish these apps for our users, lookahead and determine if the location should be provided for duplicated site names
```

Comment:
I think you should only do "groupBy === resourceType". We wouldn't want to show this for resource groups either since I don't think you can have the same named resource in one group.

Reply:
ahh good point